Automattic/wp-calypso
# Store: Stripe Connect: Add create (deferred) account flow

See #17002

To test:
* Follow the steps in #19115 to setup your environment
* **Edit: Important: Use the fix/1203-explicit-status-stripe-rest-api-errors branch when cloning woocommerce-services**
* After you've used the test bench to connect the site, navigate to http://calypso.localhost:3000/store/settings/{DOMAIN}
* Click on the Manage link for Stripe
* If you are placed on the key flow (because you have keys), click on "I want to use stripe connect"
* Leave "New Stripe Account" selected and hit Connect
* If you've already created an account for your logged in user's email, you'll get an error - that is correct behavior (see below)
* If you haven't already created an account for your logged in user's email, the dialog will update with a prompt for you to finish activating your account (see below)
* If you want to hack in a temporary email address to test this, modify the createAccount call in payment-method-stripe-setup-dialog.js to replace email with the temporary address

* Run `npm run test-client client/extensions/woocommerce/state/` and verify all tests pass

<img width="748" alt="screen shot 2017-10-30 at 10 53 09 am" src="https://user-images.githubusercontent.com/1595739/32187251-aab66ebe-bd61-11e7-8bc2-cb1a90d1e803.png">

<img width="660" alt="screen shot 2017-10-30 at 10 52 07 am" src="https://user-images.githubusercontent.com/1595739/32187249-aa9fc1f0-bd61-11e7-8a9b-7d32f4134709.png">

<img width="674" alt="screen shot 2017-10-30 at 10 54 19 am" src="https://user-images.githubusercontent.com/1595739/32187253-aacbc4a8-bd61-11e7-93d7-af5d772d2da2.png">

cc @kellychoffman @jameskoster
